Cilsil/Cil/Parsers/ExceptionParser.cs

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Cilsil.Sil;
+using Cilsil.Sil.Expressions;
+using Cilsil.Sil.Instructions;
+using Cilsil.Sil.Types;
+using Cilsil.Utils;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using System.Collections.Generic;
+
+namespace Cilsil.Cil.Parsers
+{
+    internal class ExceptionParser : InstructionParser
+    {
+        protected override bool ParseCilInstructionInternal(Instruction instruction,
```

Comment:
![image](https://user-images.githubusercontent.com/38902361/123920852-3dea5f80-d93b-11eb-9967-1b21372e985e.png)

In your translation, these conditional nodes lack exception edges to the first translation of the catch block. In the corresponding equivalent Java example, these nodes are -- is this resolved in one of your forked-repo PRs?

![image](https://user-images.githubusercontent.com/38902361/123921025-6ffbc180-d93b-11eb-8daf-f32a8859b4b4.png)